app/views/content/non-uk-teachers/promos/_adviser-promo-teach-in-england.html.erb

```diff
@@ -3,7 +3,7 @@
   <% promo.right_section(
     heading: "Get free one-to-one support"
   ) do %>
-      <p>An adviser can answer all your questions about what it's like teaching in England and what you need to get into an English classroom.</p>
+      <p>An adviser can answer all your questions about what it's like teaching in England and how to get into an English classroom. Chat through text, email or phone, as little or as often as you need.</p>
```

Comment:
@EllieNodder Minor comment (as this is already in prod) - 'how to get into an English classroom' sounds a bit odd and colloquial (through the door is surely the answer!) - worth rewording slightly?